source/_components/cover.garadget.markdown

```diff
@@ -43,3 +43,48 @@ Configuration variables:
 
 
 If provided, the **access_token** will be used, otherwise the **username** and **password** will be used to automatically generate an access token at start time.
+
+
+**Example with more detail:**
+<p class='img'>
+<img src='https://dl.dropboxusercontent.com/u/15301912/GitHub/home-assistant.github.io/images/cover_garadget_details.png' />
```

Comment:
When I am on this page: https://github.com/home-assistant/home-assistant.github.io/pull/1431
How do I know on which branch I am?